app/views/events/_form.html.erb

```diff
@@ -11,22 +11,36 @@
     </div>
   <% end %>
 
-  <div class="field">
-    <%= form.label :post_id %>
-    <%= form.text_field :post_id %>
+  <div class="input-group mb-12 mb-2">
+    <div class="input-group-prepend">
+      <%= form.label :post_id, class:"input-group-text" %>
+
+    </div>
+    <%= form.text_field :post_id, class:"form-control" %>
   </div>
 
-  <div class="field">
-    <%= form.label :date %>
-    <%= form.datetime_select :date %>
+  <div class="input-group mb-12 mb-2">
+    <div class="input-group-prepend">
+      <%= form.label :date, class:"input-group-text" %>
+
+    </div>
+    <%= form.datetime_select :date, class:"form-control" %>
   </div>
 
-  <div class="field">
-    <%= form.label :event_type_id %>
-    <%= form.number_field :event_type_id %>
+  <div class="input-group mb-12 mb-2">
+    <div class="input-group-prepend">
+      <%= form.label :event_type_id, "Jelentkező űrlap", class:"input-group-text" %>
+
+    </div>
+    <%= form.select :event_type_id, options_for_select(@event_types), include_blank: "Nincs jelentkezés" , class:"form-control" %>
```

Comment:
This way the partial less context dependendent and more reusable.
```suggestion
    <%= form.select :event_type_id, options_for_select(EventType.all.pluck(:name, :id)), include_blank: "Nincs jelentkezés" , class:"form-control" %>
```
Maybe we could split this into multiple lines.